### PR TITLE
[ACA-2474] adding instructions to the calendar for use

### PR DESCRIPTION
**Pull request by downzer0**
**Please check if the PR fulfills these requirements**

> - [ ] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)

**What is the new behaviour?**

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

please see comments from the closed PR https://github.com/Alfresco/alfresco-ng2-components/pull/5381

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -5,9 +5,9 @@
   "requires": true,
   "dependencies": {
     "@alfresco/adf-cli": {
-      "version": "3.6.0",
-      "resolved": "https://registry.npmjs.org/@alfresco/adf-cli/-/adf-cli-3.6.0.tgz",
-      "integrity": "sha512-HewDSVxz+BGqSTlooOj2jgnX5g5lNSXwsI8K+eqr55s12ku7AeX0JRFxrvVz20v/2efGMnwHHXGfF8tbdCChnA==",
+      "version": "3.7.0-beta6",
+      "resolved": "https://agora.dequecloud.com/artifactory/api/npm/dequelabs/@alfresco/adf-cli/-/adf-cli-3.7.0-beta6.tgz",
```
Comment (adinapitul) [inline package-lock.json]: I think this is not OK to be committed like this, using deque paths ....

File: lib/core/card-view/components/card-view-dateitem/card-view-dateitem.component.html [hunk 2]
```diff
@@ -32,7 +33,8 @@
         <input class="adf-invisible-date-input"
             [matDatetimepicker]="datetimePicker"
             [value]="valueDate"
-            (dateChange)="onDateChanged($event)">
+            (dateChange)="onDateChanged($event)"
+            [attr.aria-describedby]="uniqueId">
```
Comment (eromano) [inline lib/core/card-view/components/card-view-dateitem/card-view-dateitem.component.html]: can we use "'card-' + property.type + '-value-' + property.key"> as ID?

File: lib/core/styles/_a11y.scss [hunk 3]
```diff
@@ -0,0 +1,8 @@
+.adf-sr-only {
+    display: block;
```
Comment (eromano) [inline lib/core/styles/_a11y.scss]: can you move it in  card-view-dateitem.component.scss ?

**Comment by eromano** [comment]
Some unit test are failing

File: package-lock.json (hunk 1 above)
Comment (downzer0) [reply package-lock.json]: wow i'm not even sure how that happened! yeah no kidding!

File: lib/core/card-view/components/card-view-dateitem/card-view-dateitem.component.scss [hunk 4]
```diff
@@ -56,4 +56,14 @@
             }
         }
     }
+
+    .adf-sr-only {
+        display: block;
+        height: 1px;
+        width: 1px;
+        clip: rect(1 1 1 1);
+        clip: rect(1px 1px 1px 1px);
```
Comment (DenysVuika) [inline lib/core/card-view/components/card-view-dateitem/card-view-dateitem.component.scss]: clip is used 2 times

File: package.json [hunk 5]
```diff
@@ -104,6 +104,7 @@
     "@types/request": "^2.48.3",
     "adf-monaco-extension": "0.0.8",
     "adf-tslint-rules": "0.0.6",
+    "angular2-uuid": "^1.1.1",
```
Comment (DenysVuika) [inline package.json]: Homepage says "THIS PROJECT IS NOT MAINTAINED", can any other library be used? @eromano maybe we can move UUID generation helpers from e2e to the core?

File: lib/core/card-view/components/card-view-dateitem/card-view-dateitem.component.scss (hunk 4 above)
Comment (downzer0) [reply lib/core/card-view/components/card-view-dateitem/card-view-dateitem.component.scss]: there is second support for IE :(

File: package.json (hunk 5 above)
Comment (downzer0) [reply package.json]: do you guys have a library or utility that generates random IDs? pretty simple to throw together if not.

Comment (eromano) [reply package.json]: I don't think is needed anymore as the methods and the import where it is used

Comment (eromano) [reply package.json]: I don't think is needed anymore as the methods and the import where it is used

**Pull request merged**